### DISCUSSION TITLE
dokan mount using local disk type，bluescreen and code is SYSTEM_SERVICE_EXECPTION

### DISCUSSION THREAD

**Discussion by hhhhskfk** [dokan-dev/dokany#1154] [Q&A]
Blue screen appears after installing software “RavV17std.exe” and rebooting. dokan version:2.0.5.1000
software dowload url is http://download.rising.com.cn/MIDDLE12/pcv17/RavV17std-8C41ACE5/RavV17std.exe

**Comment by Olof-Lagerkvist** [comment]
> Blue screen appears after installing software “RavV17std.exe” and rebooting. dokan version:2.0.5.1000 software dowload url is http://download.rising.com.cn/MIDDLE12/pcv17/RavV17std-8C41ACE5/RavV17std.exe

Unfortunately, that installer says that it can only be installed in Chinese OS versions. Could you try to make a small reproducible example involving the driver and some user mode software that triggers this error, and/or provide more information from crashes such as dump files etc?

**Reply by hhhhskfk** [reply]
I tried to compile and collect the dump, but found that it pointed to the directory. c file while checking the code, I couldn't find any code that could cause a blue screen
![image](https://github.com/dokan-dev/dokany/assets/19182378/abcba1f3-0122-467e-acbd-7c805d6f1954)
![image](https://github.com/dokan-dev/dokany/assets/19182378/267c5fec-ed7d-4d67-af28-64c311323f51)
[minidump and pdb.zip](https://github.com/dokan-dev/dokany/files/11778703/minidump.and.pdb.zip)

**Comment by hhhhskfk** [comment]
Using fastzip software for volume compression

[RavV17std.exe.zip.001.zip](https://github.com/dokan-dev/dokany/files/11778782/RavV17std.exe.zip.001.zip)

**Comment by hhhhskfk** [comment]
[RavV17std.exe.zip.002.zip](https://github.com/dokan-dev/dokany/files/11778781/RavV17std.exe.zip.002.zip)

**Comment by Olof-Lagerkvist** [comment]
Sorry for late reply. I have not been able to reproduce this. Also, I cannot install your sample application RavV17std because the installer says "This Software can just install in Chinese Language OS".

**Comment by Olof-Lagerkvist** [comment]
![image](https://github.com/dokan-dev/dokany/assets/5272825/4675d20b-9453-4679-91a0-6dc699813f50)

**Comment by Liryna** [comment]
Closing this for https://github.com/dokan-dev/dokany/issues/1211
